packages/flutter/lib/src/rendering/box.dart

```diff
@@ -2208,6 +2403,9 @@ abstract class RenderBox extends RenderObject {
   ///    computation, call [getDistanceToActualBaseline] on the child (not
   ///    [computeDistanceToActualBaseline], the internal implementation, and not
   ///    [getDistanceToBaseline], the public entry point for this API).
+  ///
+  /// The [computeDryBaseline] method typically also needs overridding if this
+  /// method is overridden.
```

Comment:
Can you add the flip side of this also to the computeDryBaseline doc?

Reply:
I feel I should remove this line? More often than not `computeDryBaseline`  needs to be overridden, because `computeDistanceToActualBaseline` can use the child render boxes' paint offsets but the dry version won't be able to.

Reply:
removed.